packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/list/list.css

```diff
@@ -43,6 +43,11 @@
 		   https://github.com/ckeditor/ckeditor5-heading/issues/63 */
 		padding: var(--ck-list-button-padding);
 
+		&:not([class*="ck-heading_heading"]) .ck-button__label {
```

Comment:
I'm curious why we need this exception for heaings? Line heights in the heading dropdown menu have been consistent before I [introduced the regression](https://github.com/ckeditor/ckeditor5/commit/f072048026c5407467e343c116c71ff5a9d236b4#diff-89c661e74afa6f00cb58fc5bcd651f40c1750417127eae21b1d44d0f94ebf722L46) in 43.1.0. Am I missing something?

# 40.0.0

https://github.com/ckeditor/ckeditor5/assets/1099479/7d44470e-8593-459e-af30-0e9fbce83084

# 41.3.0 (regression introduced)

https://github.com/ckeditor/ckeditor5/assets/1099479/11f39646-5cb6-4d16-97a6-dcce77f1bb39

# This PR without exception

https://github.com/ckeditor/ckeditor5/assets/1099479/7d6b5045-3d50-4d8a-a484-aab0624559b9

Reply:
`line-height` is not set in `heading` plugin https://github.com/ckeditor/ckeditor5/blob/ck/6022/packages/ckeditor5-heading/theme/heading.css#L15 and setting it here can lead to more breaking changes in docs. From what I've seen, `line-height: auto` looks differently than absence of such attribute in these headers button and in terms of backward compatibility the lack of attribute is more safe.

Reply:
Without such exception, it looks like it:

![obraz](https://github.com/ckeditor/ckeditor5/assets/3949262/4ec941d8-1a5d-48b7-a4f4-e3dedf687dd1)

In our manual tests, which is not correct look, I assume.